README.md

```diff
@@ -20,12 +21,18 @@ yarn start
 - Video integration (paste youtube video link and it'll be auto-converted)
 
 ## Documentation
+
 The docs can be found [here](https://edtrio-docs.netlify.com/).
 
 ## Contribute
+
 We very much welcome any kinds of contributions! Have a look at our [issue tracker](https://github.com/schul-cloud/edtrio/issues) or [project board roadmap](https://github.com/schul-cloud/edtrio/projects/1) for inspiration.
+
 - Issue Tracker: https://github.com/schul-cloud/edtrio/issues
 - Source Code: https://github.com/schul-cloud/edtrio
 
+For some tooling explanations and guidelines, please head on over to the [docs](https://edtrio-docs.netlify.com/).
```

Comment:
hm let's maybe deeplink to the actual "Guides/Development>Tooling" section no? What do you think @Atyanar? 😊 Can do this on Monday, will add a todo for me if that's ok for you.